packages/react-core/src/components/Alert/examples/Alert.md

```diff
@@ -314,9 +304,11 @@ class AsyncLiveRegionAlert extends React.Component {
             Stop Async Info Alerts
           </button>
         </InputGroup>
-        {this.state.alerts.map(({ title, variant, isLiveRegion, key }) => (
-          <Alert variant={variant} title={title} isLiveRegion={isLiveRegion} key={key} />
+        <AlertGroup isLiveRegion aria-live="polite" aria-relevant="additions text" aria-atomic="false">
```

Comment:
It's great to show the user that they can override default values, how about since this example is very similar to the "Async alert group" example, we make a couple small changes so that they each represent an optimal use case?

So, for this example, the only thing I'd suggest changing would be to remove the redundant aria-live/relevant/atomic properties since they are already set by default in isLiveRegion and we're not overriding them. Using the "polite" announcement strategy here makes sense because they are _info_ alerts and not as critical to the users immediate attention. Also, consider increasing the timeout from 1500ms to ~4500 so that each announcement finishes before the next starts. That way the example is less rappid fire and more realistic.

Then, for AlertGroup "[Async](https://github.com/jessiehuff/patternfly-react/blob/fix/live-alert-group/packages/react-core/src/components/AlertGroup/examples/AlertGroup.md#async-alert-group)" example we could add `aria-live="assertive"` as an override and switch those alerts from info to danger (also increasing timeout for those). This would perfectly simulate how the alert might be announced to the user in a real app setting. Using the "assertive" announcement strategy here makes sense because they are _danger_ alerts which are more likely to require the users immediate attention from a UX point of view.

Beyond that, the only other thing I can think of that would useful would be to add a small description of what's going on in each of these examples and why the different a11y-related properties were used and what the end result is on the AX.